### PR TITLE
updated the config to support servlets.

### PR DESCRIPTION
**Pull request by shankarkc**
updated the config to support servlets. Removed the code that was adding empty servlet tag when servlet was not required.
Added null checks
Changed type to array
@smccarthy  can you pls merge this pull request?

### CODE REVIEW

**Comment by shankarkc** [comment]
@smccarthy can you pls merge this pull request? Once done I can resolve the defect 
https://github.com/groupon/Selenium-Grid-Extras/issues/286
Thanks
Shankar KC

File: SeleniumGridExtras/src/main/java/com/groupon/seleniumgridextras/config/GridNode.java [hunk 1]
```diff
@@ -34,8 +32,7 @@
 
   // Selenium 3 has values at top level, not in "configuration"
   private String proxy;
-  
-  private List<String> servlets ;
+  ArrayList<String> servlets = new ArrayList<String>();
```
Comment (smccarthy) [inline SeleniumGridExtras/src/main/java/com/groupon/seleniumgridextras/config/GridNode.java]: Should this be private ?

**Comment by smccarthy** [comment]
@shankarkc Thanks for this PR. I had two minor comments about why servlets was made non-private, and why change from List to ArrayList ?

File: SeleniumGridExtras/src/main/java/com/groupon/seleniumgridextras/config/GridNode.java (hunk 1 above)
Comment (shankarkc) [reply SeleniumGridExtras/src/main/java/com/groupon/seleniumgridextras/config/GridNode.java]: Thanks @smccarthy I missed it.  I will change it and update the pull request. Regarding the arraylist... As i have used json helper it returns arraylist type. So i will retain it and update visibility to private.

Comment (smccarthy) [reply SeleniumGridExtras/src/main/java/com/groupon/seleniumgridextras/config/GridNode.java]: Sounds good. Thank you. I'll merge after your update.

**Comment by shankarkc** [comment]
As suggested i updated variable visibility to private. As I have used Gson helper that returnsAarrayList i have used ArrayList instead of list. @smccarthy pls merge the pull request.
Thanks you
Shankar KC

**Pull request merged**